.github/workflows/build.yaml

```diff
@@ -6,12 +6,14 @@ on:
       - 'docs/**'
       - 'tools/**'
       - '.github/dependabot.yml'
+      - 'bottlerocket/tests/workload/**'
```

Comment:
I wonder if we should add a new make target to do any validation on the workload tests. Maybe just a verification that the docker file builds. Something to ponder for the future.